instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/TracingFilter.java

```diff
@@ -56,11 +57,12 @@ public void setTracing(Tracing tracing) {
     tracer = tracing.tracer();
     extractor = tracing.propagation().extractor(GETTER);
     injector = tracing.propagation().injector(SETTER);
+    isInit = true;
   }
 
   @Override
   public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
-    if (tracer == null) return invoker.invoke(invocation);
+    if (isInit == false) return invoker.invoke(invocation);
```

Comment:
since about extension loader. how about.. `if (!loaded) return invoker.invoke(invocation);`

Reply:
yeh, it also works.